src/components/skeletons/blogSkeleton.tsx

```diff
@@ -56,3 +56,15 @@ export const BlogCardSkeleton = () => {
     </div>
   );
 };
+
+export const BlogCardListSkeleton = () => {
+  return (
+    <div className='w-full space-y-6 md:space-y-8'>
+      {Array(4)
```

Comment:
Array(4).fill() will create a new array everytime, just repeat the skeleton 4 times